PowerEditor/src/ScintillaComponent/AutoCompletion.cpp

```diff
@@ -673,6 +682,7 @@ bool AutoCompletion::showWordComplete(bool autoInsert)
 
 	_pEditView->execute(SCI_AUTOCSETSEPARATOR, WPARAM(' '));
 	_pEditView->execute(SCI_AUTOCSETIGNORECASE, _ignoreCase);
+	_pEditView->execute(SCI_AUTOCSETCASEINSENSITIVEBEHAVIOUR, _ignoreCase);

```

Comment:
@mpheath Any example how this line change actual behavior?

Reply:
It may have no real impact there though the setting `SCI_AUTOCSETCASEINSENSITIVEBEHAVIOUR`  needs to be reset on the point of showing the calltip, else it inherits the last setting like is in path completion which is always sets it to true.

https://github.com/notepad-plus-plus/notepad-plus-plus/blob/6a8a446eb61f96b00f74ae5baa0894ff16e7b18f/PowerEditor/src/ScintillaComponent/AutoCompletion.cpp#L612

Word completion seems to be always be case sensitive, so the `SCI_AUTOCSETIGNORECASE` set to ignore case seems a strange as variable though it is there. And since it is there and `SCI_AUTOCSETCASEINSENSITIVEBEHAVIOUR` may need to be set to what that autocomplete function requires, it has been inserted there.